src/main/java/org/primefaces/component/datatable/export/DataTableCSVExporter.java

```diff
@@ -66,8 +65,6 @@ public void doExport(FacesContext context, DataTable table, ExportConfiguration
         try (OutputStreamWriter osw = new OutputStreamWriter(getOutputStream(), exportConfiguration.getEncodingType());
             PrintWriter writer = new PrintWriter(osw);) {
 
-            ExternalContext externalContext = context.getExternalContext();
-
```

Comment:
Dead variable according to Sonar.

Reply:
Pretty sure I haven't worked on the DataTableCSVExporter, but yes, the variable `externalContext` is not used anywhere.

Reply:
Yes, should be removed.